OSSM-8953 Add update approval strategy / channel guidance to operator install procedure (#93016)
openshift/openshift-docs: modules/ossm-installing-operator.adoc

. When the prompt that discusses the community operator appears, click *Continue*.

. Click *Install*.

. On the *Install Operator* page, perform the following steps:

.. Select *All namespaces on the cluster (default)* as the *Installation Mode*. This mode installs the Operator in the default `openshift-operators` namespace, which enables the Operator to watch and be available to all namespaces in the cluster.

.. Select *Automatic* as the *Approval Strategy*. This ensures that the {olm-first} handles the future upgrades to the Operator automatically. If you select the *Manual* approval strategy, {olm} creates an update request. As a cluster administrator, you must then manually approve the {olm} update request to update the Operator to the new version.

.. Select an *Update Channel*.
+
* Choose the *stable* channel to install the latest stable version of the {SMProductName} 3 Operator. It is the default channel for installing the Operator.
+

> * TTo install a specific version of the {SMProductName} 3 Operator, choose the corresponding `stable-<version>` channel. For example, to install the {SMProductName} Operator version 3.0.x, use the *stable-3.0* channel.

Reviewer: ```suggestion
* To install a specific version of the {SMProductName} 3 Operator, choose the corresponding `stable-<version>` channel. For example, to install the {SMProductName} Operator version 3.0.x, use the *stable-3.0* channel.
```

. Click *Install* to install the Operator.

> . Click *Operators* -> *Installed Operators* to verify that the {SMProductShortName} Operator is installed. `Succeeded` should appear in the *Status* column.

Reviewer: .Verification

Click *Operators* -> *Installed Operators* to verify that the {SMProductShortName} Operator is installed. `Succeeded` should show in the *Status* column.

> . Verify the {SMProductShortName} Operator is version 3.0, and click *Install*.

Reviewer 2: Maybe just "Click Install" would make more sense, and selecting version should be added to you new steps? (because if the user selects different update channels, I would expect that the list of possible versions will change and the version field would probably update automatically to the latest option)

. On the *Install Operator* page:

.. Select *All namespaces on the cluster (default)* for the *Installation Mode*. This mode installs the Operator in the default `openshift-operators` namespace, which enables the Operator to watch and be made available to all namespaces in the cluster.

.. Select *Automatic* for the *Approval Strategy*. This ensures that the future upgrades to the Operator are handled automatically by the {olm-first}. If you select the *Manual* approval strategy, {olm} creates an update request. As a cluster administrator, you must then manually approve the {olm} update request to update the Operator to the new version.

.. Select an *Update Channel*.
+

> * The channel enables installation of the most recent stable version of the {SMProductName} 3 Operator. Currently, it is the default channel for installing the {SMProductName} 3 Operator.

Reviewer 2: And maybe this can be additionally made more action-oriented:
```suggestion
* Choose the *stable* channel to install the latest stable version of the {SMProductName} 3 Operator. It is the default channel for installing the Operator.
```

> * The latest channel enables installation of the most recent stable version of the {SMProductName} 3 Operator. Currently, it is the default channel for installing the {SMProductName} 3 Operator.

Reviewer 3: We are not using the `latest` channel. There is `stable` channel and `stable-<version>` for each minor version.

Author: @FilipB I will remove the word "latest" to avoid confusion. It can be rephrased to: "The channel enables installation of the most recent stable version of the {SMProductName} 3 Operator."

The point for "stable-<version>" is used in the second bullet point.

+

> * To install a specific version of the {SMProductName} 3 Operator, cluster administrators can use the corresponding `stable-<version>` channel. For example, to install the {SMProductName} Operator version 3.0.x, you can use the pipelines-3.0 channel.

Reviewer 2: Is the "pipelines" correct here? Based on the description, I thought we were talking about `stable-<version>` structure.
```suggestion
* To install a specific version of the {SMProductName} 3 Operator, cluster administrators can use the corresponding `stable-<version>` channel. For example, to install the {SMProductName} Operator version 3.0.x, use the *stable-3.0* channel.
```

Reviewer 2: And, optionally, to possibly make it more action-oriented as the previous bullet, if you would want to do so:
```suggestion
* To install a specific version of the {SMProductName} 3 Operator, choose the corresponding `stable-<version>` channel. For example, to install the {SMProductName} Operator version 3.0.x, use the *stable-3.0* channel.
```

Reviewer 2: Note that I think specifying cluster admins here is redundant as this is implied thanks to the cluster-admin role being a prerequisite.
